nettle/plan.sh

```diff
@@ -1,12 +1,12 @@
 pkg_name=nettle
 pkg_origin=core
-pkg_version="3.4.1"
-pkg_description="a low-level cryptographic library"
+pkg_version=3.5.1
+pkg_description="A low-level cryptographic library"
 pkg_upstream_url="https://www.lysator.liu.se/~nisse/nettle/"
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('LGPL-3.0' 'GPL-2.0' 'GPL-3.0')
```

Comment:
SPDX has updated their short-strings for these license since this plan was created. Would you mind updating the license strings to match their new format?  https://spdx.org/licenses/

Reply:
And what if I *do* mind?

![tenor-109704642](https://user-images.githubusercontent.com/24568/60631201-518c8980-9e39-11e9-9c2a-7fdd1123dd4a.gif)

Joking, I'm on it :)

Reply:
@smacfarlane Done - But please check them to make sure. Its weird in that it specifies 3 particular licensing schemes.